src/jni/src/datamanlite/NebulaSchemaProvider.h

```diff
@@ -0,0 +1,86 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef DATAMAN_CODEC_NEBULASCHEMAPROVIDER_H_
+#define DATAMAN_CODEC_NEBULASCHEMAPROVIDER_H_
+
+#include <unordered_map>
+#include <vector>
+#include "datamanlite/SchemaProviderIf.h"
+#include "datamanlite/Slice.h"
+
+namespace nebula {
+namespace dataman {
+namespace codec {
+
+class NebulaSchemaProvider : public SchemaProviderIf {
+public:
+    class SchemaField final : public SchemaProviderIf::Field {
+    public:
+        SchemaField(std::string name, ValueType type)
+            : name_(std::move(name))
+            , type_(type) {}
+
+        const char* getName() const override {
+            return name_.c_str();
+        }
+
+        const ValueType& getType() const override {
+            return type_;
+        }
+
+        bool isValid() const override {
```

Comment:
If there is nothing to check, could we remove `isValid` ?

Reply:
ditto